helm/nessie/values.yaml

```diff
@@ -141,6 +141,27 @@ advancedConfig:
 #      console.format: "%d{HH:mm:ss} %s%e%n"
 #      category."org.projectnessie".level: DEBUG
 
+# Reverse Proxy Settings
+# ----------------------
+#
+# These config options are mentioned only for documentation purposes. Consult the
+# Quarkus documentation for "Running behind a reverse proxy" and configure those
+# depending on your actual needs.
+#
+# See https://quarkus.io/guides/http-reference#reverse-proxy
+#
+# Do NOT enable these option unless your reverse proxy (for example istio or nginx)
+# is properly setup to set these headers but also filter those from incoming requests.
+#
+#  quarkus:
+#   http:
+#     proxy:
+#       proxy-address-forwarding=true
+#       allow-x-forwarded=true
+#       enable-forwarded-host=true
+#       enable-forwarded-prefix=true
```

Comment:
The prefix does not work to the best of my knowledge. Did you manage to have a working example with a prefix in the reverse proxy? I'd prefer to note that explicitly if it is indeed not working well.

https://github.com/projectnessie/nessie/pull/8674#issuecomment-2145309722

Reply:
Let me check

Reply:
Works with a simple fix now. Updated the example to use `/nessie/` as the prefix.